adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
Why is this using a floating point multiplication? You can get the right (and more accurate) result with an integer `data.timestamp / 1000`.

Also, `data.timestamp` is `int64_t`. Negative timestamps don't make sense, but casting them to unsigned will probably make even less sense..